testsuite/run_sets/secondary.yml

```diff
@@ -65,4 +65,5 @@
 - features/secondary/proxy_cobbler_pxeboot.feature
 - features/secondary/srv_restart.feature
 - features/secondary/min_timezone.feature
+- features/secondary/proxy_as_pod_basic_tests.feature
```

Comment:
did you forget `testsuite/features/core/proxy_register_as_pod.feature` or did you put it somewhere else I may have missed?

Reply:
It's explained in the card, but not in the PR.
I will do it. Basically I prepared a feature in advance, we will not use it yet, only when the new proxy supports all the features in the secondary stage, then we will enable the feature in core, and disable that one.